lib/node_modules/@stdlib/stats/base/dists/rayleigh/cdf/README.md

```diff
@@ -103,7 +103,9 @@ y = cdf( 2.0, 0.0 );
 Returns a function for evaluating the [cumulative distribution function][cdf] of a [Rayleigh][rayleigh-distribution] distribution with parameter `sigma` (scale parameter).
 
 ```javascript
-var myCDF = cdf.factory( 0.5 );
+var y;
+var myCDF;
+myCDF = cdf.factory( 0.5 );
 y = myCDF( 1.0 );
```

Comment:
Let's not change this. For example code, we typically try to initialize variables with their values instead of hoisting the `var` declarations to the top:

```suggestion
var myCDF = cdf.factory( 0.5 );
var y = myCDF( 1.0 );
```

Reply:
Ya but actually previously variable Y is not defined and getting error in Example run for this code .